requirements/production.txt

```diff
@@ -2,5 +2,5 @@ docutils>=0.12
 Jinja2>=2.7.3
 MarkupSafe>=0.23
 Pygments>=1.6
-Sphinx>=3.0,<3.4
+Sphinx>=3.0,<3.3
```

Comment:
If you change it here, can you also change it in `breathe/setup.py`?

Reply:
Done